hidimstat/test/test_generate_knockoff.py

```diff
@@ -20,18 +20,45 @@ def test_estimate_distribution():
     assert mu.size == p
     assert Sigma.shape == (p, p)
 
-    mu, Sigma = _estimate_distribution(X, cov_estimator="graph_lasso")
+
+def test_estimate_distribution_lasso():
+    SEED = 42
```

Comment:
capitalized variables are global in a module (constants), so it does not make sense to define such a variable in a function.